src/app/services/session-storage.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from "@angular/core";
+import AppValues from "../common/app.values";
+
+@Injectable({
+    providedIn: "root",
+})
+export class SessionStorageService {
+    public get sidePanelVisibility(): string | null {
```

Comment:
I think you should encapsulate conversion to boolean here, and a default value when key is unavailable.
This looks bad:

```
this.showSidePanel = JSON.parse(visiblePanel) as boolean;
```

I just want to call a service hiding session storage, and obtain my boolean value as is.

Reply:
Moved conversion to boolean into service